src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -559,7 +515,7 @@ const formConfig = {
                               '1',
                               '2',
                               '3',
-                              // '4',
+                              // '4', // TODO: Confirm this should be taken out
```

Comment:
All of these doc types are temporary and borrowed from another form actually as we are working with @goldenmeanie to produce the relevant list.

Reply:
We may remove this document type altogether if it's not useful when we upload. (Sounded like only the name was because whomever reviews these doesn't trust the document type...but perhaps that was just for additional evidence?)